docs/guide/optimizer.md

```diff
@@ -59,35 +59,35 @@ class Song(models.Model):
 ```
 
 ```{.python title=types.py}
-import strawberry
-import strawberry.django
+from strawberry import auto
+import strawberry_django
 
-@strawberry.django.type(Artist)
+@strawberry_django.type(Artist)
 class ArtistType:
     name: auto
     albums: list["AlbumType"]
-    albums_count: int = strawberry.django.field(annotate=Count("albums"))
+    albums_count: int = strawberry_django.field(annotate=Count("albums"))
 
 
-@strawberry.django.type(Album)
+@strawberry_django.type(Album)
 class AlbumType:
     name: auto
     release_date: auto
     artist: ArtistType
     songs: list["SongType"]
 
 
-@strawberry.django.type(Song)
+@strawberry_django.type(Song)
 class SongType:
     name: auto
     duration: auto
     album_type: AlbumType
 
 
-@strawberry.type
+@strawberry_type
```

Comment:
Also, this should still be `@strawberry.type`

Reply:
I don't know why I did this!